sourced/ml/__main__.py

```diff
@@ -57,6 +57,28 @@ def add_parser(name, help):
     add_engine_args(repos2df_parser)
     add_feature_args(repos2df_parser)
     # ------------------------------------------------------------------------
+    repos2ids_parser = subparsers.add_parser(
+        "repos2ids", help="Convert source code to a bag of identifiers.")
+    repos2ids_parser.set_defaults(handler=repos2ids_entry)
+    add_engine_args(repos2ids_parser)
+    repos2ids_parser.add_argument(
+        "-r", "--repositories", required=True,
+        help="The path to the repositories.")
+    repos2ids_parser.add_argument(
+        "-o", "--output", required=True,
+        help="[OUT] The path to the output csv file of identifiers, compressed in gz format.")
```

Comment:
output **CSV** file **with** identifiers

You should not compress it unless the file name ends with `.gz`